sw/device/lib/testing/otbn_testutils.h

```diff
@@ -0,0 +1,68 @@
+// Copyright lowRISC contributors.
+// Licensed under the Apache License, Version 2.0, see LICENSE for details.
+// SPDX-License-Identifier: Apache-2.0
+
+#ifndef OPENTITAN_SW_DEVICE_LIB_TESTING_OTBN_TESTUTILS_H_
+#define OPENTITAN_SW_DEVICE_LIB_TESTING_OTBN_TESTUTILS_H_
+
+#include "sw/device/lib/runtime/otbn.h"
+#include "sw/device/lib/testing/rv_plic_testutils.h"
+
+#include "hw/top_earlgrey/sw/autogen/top_earlgrey.h"
+
+/**
+ * OTBN test utils context structure.
+ *
+ * Use `otbn_testutils_init_irq()` to initialize.
+ */
+typedef struct isr_context {
+  /**
+   * The otbn dif to access the hardware.
+   */
+  dif_otbn_t *dif;
+
+  /**
+   * The plic dif to access the hardware.
+   */
+  dif_rv_plic_t plic;
```

Comment:
Why not a pointer?

Reply:
A pointer here would make this service more bureaucratic to be used by the test. As the objective of this service is to wrap  the plic peripheral, it does not make sense to have the `dif_rv_plic_t` allocated by the test. 
Does it make sense?